staff/joel-molero/cv/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Curriculum Vitae</title>
+</head>
+<body>
+    <header>
+        <h1>Joel Molero</h1>
+        <br>
+        <address>Address: Spain, Barcelona, Febot street, Nº24, 2-1 
```

Comment:
address should be in separated p tags inside the address tag

Reply:
As in:
``` 
<address>
<p>...</p>
</address>```
?

Reply:
yes m8